src/search/landmarks/landmark_factory_h_m.cc

```diff
@@ -1054,12 +1002,13 @@ static shared_ptr<LandmarkFactory> _parse(OptionParser &parser) {
         "h^m Landmarks",
         "The landmark generation method introduced by "
         "Keyder, Richter & Helmert (ECAI 2010).");
-    parser.document_note(
-        "Relevant options",
-        "m, reasonable_orders, conjunctive_landmarks, no_orders");
     parser.add_option<int>(
         "m", "subset size (if unsure, use the default of 2)", "2");
-    _add_options_to_parser(parser);
+    parser.add_option<bool>(
+        "conjunctive_landmarks",
+        "keep conjunctive landmarks",
```

Comment:
"keeping" means they are always computed and discarded on demand, right? Then I would suggest to call this option `keep_conjunctive_landmarks` or even better, `discard_conjunctive_landmarks`, to match the other discarding option and the name of the method in the code.